### ISSUE TITLE
Strange Asterisk expansion

### ISSUE THREAD

**Issue by fab1an** [natesilva/javascript-eslint.tmbundle#4]
When I put this in a file:
````
/*** Comment */
````

I get:

` at line 1 position 2:  Expected space or tab after /Applications /Library /Network /System /User Information /Users /Volumes /bin /cores /dev /etc /home /installer.failurerequests /lost+found /net /opt /private /sbin /tmp /usr /var in comment.`

**Comment by natesilva** [comment]
Yikes, that’s not a nice error. Try the new version 2.0.0 of the bundle. I can’t reproduce this problem.

**Comment by fab1an** [comment]
Still there:

![screen shot 2015-11-04 at 10 42 16](https://cloud.githubusercontent.com/assets/55547/10934539/bf22038a-82e0-11e5-8b73-022a65b73b47.png)

My `.eslintrc` (in home-directory)

```javascript
{
  "extends": "eslint:recommended",
  "extends": "standard",
  "ecmaFeatures": {
    "jsx": true,
    "modules": true
  },
  "env": {
    "browser": true,
    "es6": true
  },
  "plugins": [
    "react"
  ],
  "rules": {
    "semi": 0,
    "quotes": 0,
    "no-multi-spaces": 2,
    "padded-blocks": 0,
    "one-var": 0,
    "standard/object-curly-even-spacing" : 2,
    "space-before-function-paren": 0,
    "no-trailing-spaces" : 0,
    "no-underscore-dangle": 0,
    "no-undef": 2,
    "comma-dangle": 0,
    "standard/array-bracket-even-spacing": 0,
    "indent": [2, 2, {"VariableDeclarator": {"var": 2, "let": 2, "const": 3}, "SwitchCase": 1}],
    "no-catch-shadow": 2,
    "no-shadow": 2,
    "no-undefined": 2,
    "no-unused-vars": [2, {"varsIgnorePattern": "(^L$|^cls$|^Optional$|^checkState$|^checkNotNull$|^checkType$|^Map$|^List$|^React$|^fromJS$|^Immutable$|^deepAccess$)"}],
    "no-use-before-define": 0,
    "react/jsx-uses-vars":2,
  }
}
```

**Comment by fab1an** [comment]
Please reopen this ticket.

**Comment by natesilva** [comment]
Thank you. With that config I was able to reproduce the problem and fixed it in 2.0.1.

**Comment by fab1an** [comment]
You're welcome. Btw.: http://www.shellcheck.net/
